src/Storage/Storage.Management/help/Enable-AzStorageBlobDeleteRetentionPolicy.md

```diff
@@ -47,6 +47,13 @@ Enabled Days
 
 This command enables delete retention policy for the Blob service, and set deleted blob retention days to 4.
 
+### Example 2
```

Comment:
Why we need this example, this can be covered by example #1.

Reply:
This example doesn't have the -PassThru option. Doesn't that make it a new example?